includes/payment-methods/class-afterpay-payment-method.php

```diff
@@ -72,8 +72,10 @@ public function __construct( $token_service ) {
 	 * Returns payment method title.
 	 *
 	 * @param string|null $account_country Country of merchants account.
-	 * @param array|false $payment_details Optional payment details from charge object.
+	 * @param array|false $payment_details Payment details from charge object. Not used by this class.
 	 * @return string|null
+	 *
+	 * @phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
 	 */
 	public function get_title( string $account_country = null, $payment_details = false ) {
```

Comment:
https://github.com/Automattic/woocommerce-payments/issues/8657 suggests we apply the inline only in the parent classes that have unused variables, but this child class outputs `ParamNameMismatch` phpcs error if we resolve the problem through renaming the variable to `$unused_payment_details`, so I used the inline comment here as well.